client/app/scripts/superdesk-templates/templates.js

```diff
@@ -156,8 +157,37 @@
                 desks.initialize()
                 .then(function() {
                     $scope.desks = desks.desks;
+
+                    /*
+                     * Returns desk name
+                     */
+                    $scope.getTemplateDesk = function getTemplateDesk(template) {
+                        return _.find($scope.desks._items , {_id: template.template_desk});
+                    };
+
+                    /*
+                     * Returns stage name
+                     */
+                    $scope.getTemplateStage = function getTemplateStage(template) {
+                        return _.find(desks.stages._items , {_id: template.template_stage});
+                    };
```

Comment:
Is there a specific reason that these two methods are defined inside the promise resolve handler? They can (and probably should) be simply defined outside of it, just like the rest.